File: src/ejemplos/ejemplo1.js
import sys from "sys"
import glob from "glob"
import serial from "serial"
import list_ports from "serial/tools/list_ports"
import { pprint } from "pprint"
import uuid from "uuid"
function getserialport() {
    let ports = serial.tools.list_ports.comports()
    if (ports.length < 1) {
        console.log("No serial port connected")
        process.exit(1)
    }
    return ports[0]
}
if (__filename == '__main__') {
    let port = getserialport()
    let s = serial.Serial(port.device, 115200)
    let uid = uuid.uuid4()
    console.log("Waiting for card")
    s.close()
}

File: src/ejemplos/ejemplo1.py
import sys
import glob
import serial
import serial.tools.list_ports
from pprint import pprint


import uuid


def getserialport():
    ports = serial.tools.list_ports.comports()

    if len(ports) < 1:
        print("No serial port connected")
        exit(1)
        
    return ports[0]

if __name__ == '__main__':
    # print(serial_ports())

    port = getserialport()
    s = serial.Serial(port.device, 115200)

    uid = uuid.uuid4()


    print("Waiting for card")


    s.close()